src/ItemGrid.cpp

```diff
@@ -345,14 +352,14 @@ int cItemGrid::AddItem(cItem & a_ItemStack, bool a_AllowNewStacks, int a_Priorit
 			// All items fit
 			return a_ItemStack.m_ItemCount;
 		}
-	}  // for i - m_Slots[]
+	}  // for i - Slots[]
```

Comment:
What is the point of this comment change? Does this code not still iterate over `m_Slots`?

Reply:
There used to be an intermediate variable that's not there any more.
Fixed.